docs/user/gcp/customization.md

```diff
@@ -73,6 +80,13 @@ compute:
       osDisk:
         diskType: pd-standard
         diskSizeGB: 128
+        encryptionKey:
+          kmsKey:
+            name: worker-key
+            keyRing: openshift-machine-keys
+            location: global
+            projectID: openshift-dev-installer
```

Comment:
I think it would be more clear to keep this section as a minimal example of customized machine pools and move the encryption key parts to their own section with a very brief intro sentence, similar to https://github.com/openshift/installer/pull/4468/files#diff-0a5c8c10127bbbc00d3eee65e22fa9b95666c829243b576d23ab64e2aceb2482L99

I think that would be better but I think this is good too.
/lgtm